rootdir/ueventd.qcom.rc

```diff
@@ -280,5 +280,5 @@
 # IR
 /dev/lirc0                0660   system     system
 /dev/peel_ir              0660   system     system
-/dev/spidev7.1            0660   system     system
```

Comment:
I rather prefer chmod this in init.rosy.rc as we have two nodes for spi interface control